guide/lib/helpers/link_helpers.rb

```diff
@@ -6,11 +6,14 @@ def anchor_id(caption)
   end
 
   module LinkHelpers
+    COMPONENTS_RE = %r{(?<url>/components/.*)\..*}.freeze
+
     def component_links
-      {
-        "Alerte" => "/components/alert",
-        "Accordéon" => "/components/accordion"
-      }
+      @items.map do |item|
+        data = item.attributes[:filename].match(COMPONENTS_RE)
+
+        [item[:title], data[:url]] if data.present?
```

Comment:
nice! If I understand correctly the `title` attribute is set through frontmatter metadata. Does it have a default that matches the filename ? 

cf https://nanoc.app/doc/tutorial/#edit-the-home-page

otherwise, did you set it on each existing component?

Reply:
Yes, I just re-used the `title` frontmatter in the `guide/content/components/` page. It was set everywhere so I didn't do much.